.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+
+notifications:
+      email: false
+
+python:
+    - "3.4"
+    - "3.5"
+    - "3.6"
```

Comment:
should include 3.7

Reply:
I tried but it's not supported yet. still an open issue on travis:
https://github.com/travis-ci/travis-ci/issues/9815